source/includes/fact-more-than-500-orgs.rst

```diff
@@ -0,0 +1,5 @@
+.. note::
+
+   If you have access to more than 500 organizations, you must
+   type the organization ID or the organization name to filter the
+   results before using the arrows to select an organization.
```

Comment:
**Style Nit:** See [Clarify Gerund and Participles](https://www.mongodb.com/docs/meta/style-guide/writing/clarify-gerunds-participles/)

```suggestion
   results before you use the arrows to select an organization.
```